downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
+
+. Under *Image Pull Policy*, click on the radio button to select *Always*, *Never*, or *IfNotPresent*.
+. Define the *Image Pull Secrets*
+.. Click on the angle bracket to show the *Add Image Pull Secret*.
+.. Click on the *Add Image Pull Secret* and enter the *Value* in the field.
```

Comment:
```suggestion
.. Click *Add Image Pull Secret* and enter a *Value* .
```
Another suggestion to make things concise. Just a question. Are there parameters for the value field? Will the customer know what the values should be? Just thinking through what a user would expect.